docs/api-references/user.md

```diff
@@ -0,0 +1,127 @@
+## User API
+
+### `GET` User
+```
+/api/user/
+```
+Get information about a current user login.
+
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
+```
+#### Sample Response
+```json
+{
+    "data": {
+                "id": 74,
+                "full_name": "dungvan2512",
+                "email": "dungvan@test.email.com",
+                "birthday": null,
+                "gender": 0,
+                "address": null,
+                "phone_number": null,
+                "image": "http://foodmarket.com/images/users/default.jpg",
+                "is_admin": 1,
+                "is_active": 0,
+                "created_at": "2017-08-23 04:13:40",
```

Comment:
If this API don't need to use `created_at `, `updated_at`, `deleted_at`. You don't need to return them. Select only value feature to want to use.